.github/workflows/alpine-release.yml

```diff
@@ -0,0 +1,34 @@
+name: Alpine Linux release
+on: 
+  push:
+    tags:
+      - v*
```

Comment:
If I'm reading this correctly (i.e. using tags that match `v*`) then it's wrong.  We haven't been consistent so far with the tag naming, but I'll stick to just using the version number from now on (i.e. 7.3.0p0, 7.4.0p0, etc...) without any prefix.

Reply:
Sure, that works. I saw the tag drop down in the Github UI and saw a couple vXXX tags, so I assumed thats what it was.

Reply:
Fixed.